docs/native/privacy.md

```diff
@@ -0,0 +1,33 @@
+# Privacy
+
+Altis Analytics by default does not record visitor IP addresses to help protect privacy.
```

Comment:
Should we include additional information here about obligations re cookie law/GDPR?

Reply:
Yeah eventually but I'm not completely sure what those are or should be right now.

Reply:
OK, let's ship with this for now, and I'll come back and write up something based on the engineering handbook guidance.